package.json

```diff
@@ -14,12 +13,10 @@
   },
   "devDependencies": {
     "babel-core": "^6.26.0",
-    "babel-eslint": "^10.0.1",
+    "babel-eslint": "9.0.0",
```

Comment:
You downgraded the version of eslint? Was that intentional?

Reply:
Yep - `create-react-app@2` now requires exact versions of eslint. Not awesome but also not terrible (not sure what that version got us). For this and a few other reasons I don't actually think CRA is a great fit anymore for the docs so will address in further PRs.